_sass/blocks/_landing.scss

```diff
@@ -0,0 +1,85 @@
+.landing-section_top,
+.landing-section_open,
+.landing-section {
+	padding-bottom: $base-padding;
+	padding-top: $base-padding;
+}
+
+.landing-section {
+	@include respond-to(medium-up) {
+		@include span-columns(5.5);
+
+		border-bottom: none;
+
+		&:nth-child(even) {
+			@include shift(1);
+		}
+	}
+
+	border-bottom: $thin-border-size solid $mid-gray;
+}
+
+.landing-section_open {
+	@extend .landing-section;
+
+	&.oil-gas {
+		background: url('../img/revenues/cards/oil-gas.png') bottom right no-repeat;
+	}
+
+	&.minerals {
+		background: url('../img/revenues/cards/minerals.png') bottom right no-repeat;
+	}
+
+	&.coal {
+		background: url('../img/revenues/cards/coal.png') bottom right no-repeat;
+	}
+
+	&.renewables {
+		background: url('../img/revenues/cards/renewables.png') bottom right no-repeat;
+	}
+
+	&.oil-gas,
+	&.coal,
+	&.minerals,
+	&.renewables {
+		background-size: 100%;
+		height: 310px;
+
+		.landing-description {
+			@include span-columns(6);
+		}
+	}
+
+	.landing-heading {
+		display: block;
+	}
+
+}
+
+.landing-section_category {
+  margin-top: $base-padding;
```

Comment:
Properties should be ordered border-bottom, margin-bottom, margin-top, padding-bottom